doc/requirements/eviction

```diff
@@ -0,0 +1,27 @@
+---
+group: Eviction
+---
+
+Eviction controlls a process of invalidating clean data from cache.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Eviction priorities
+id: priorities
+---
+
+The product shall shall perfrom eviction based on the priority information
+associated with IO classes.
+
+The product shall evict in following order:
+
+1. IO classes with excedeed occupancy limit.
+2. Lower priority IO classes (omit pinned IO Classes)
```

Comment:
Like mentioned in other review: is "pinning" a term that is described anywhere in user documentation?

Reply:
I proposed definition of pinned ioclass in ioclass requirements.